deeplabcut/gui/labeling_toolbox.py

```diff
@@ -193,14 +194,16 @@ def drawplot(self, img, img_name, itr, index, bodyparts, cmap, keep_view=False):
             self.toolbar = NavigationToolbar(self.canvas)
         return (self.figure, self.axes, self.canvas, self.toolbar)
 
-    def getColorIndices(self, img, bodyparts):
-        """
-        Returns the colormaps ticks and . The order of ticks labels is reversed.
-        """
-        im = cv2.imread(img)
-        norm = mcolors.Normalize(vmin=0, vmax=np.max(im))
-        ticks = np.linspace(0, np.max(im), len(bodyparts))[::-1]
-        return norm, ticks
+
+def getColorIndices(img, bodyparts):
+    """
+    Returns the colormaps ticks and . The order of ticks labels is reversed.
+    """
```

Comment:
@jeylau can I suggest updating the docstring? Do you think updating it is worth the effort?

```suggestion
def getColorIndices(img, bodyparts):
    """Returns the colormaps ticks and ?.

    Parameters
    ----------
    img: numpy array of shape?
        The image from which color information is extracted?
    bodyparts: type?
        ?

    Returns
    -------
    norm: type?
        ?
    ticks: numpy array?
        ?
        Note that the order of ticks labels is reversed. (i'm not sure what is being reversed here and why)
    """
```

Reply:
Hey @rahulporuri, don't worry about these functions. They are very internal to the GUI, which is currently being rewritten anyhow 😌

Reply:
Ahh. Thanks for the clarification. Please ignore my comment.